### ISSUE TITLE
New YA list to scrape!

### ISSUE THREAD

**Issue by jenny8lee** [writingatlas/change-requests#23]
@eric-zhizu! Great list to scrape!

https://bookriot.com/ya-short-stories/

**Comment by eric-zhizu** [comment]
Done.
https://airtable.com/tblQybfjJb3m8Tldz/viw6Pse0497wB8KAf?blocks=hide

Question:
What "Honor" should we give these stories? (To keep track of them when we put them in the Master table)

**Comment by jenny8lee** [comment]
"Bookriot Pick"

**Comment by eob** [comment]
@eric-zhizu @jenny8lee is there anything special I need to do to get these on the site?

**Comment by eric-zhizu** [comment]
No, it's been already added to the main table. This should be closed, but I'm not sure how to close it.
